wg21_paper_tracker/cloud_run_job/converters/openai_converter.py

```
@@ -0,0 +1,269 @@
+        List of PIL Image objects.
+    """
+    if not PDF2IMAGE_AVAILABLE:
+        logger.error("pdf2image is not available")
+        return []
+
+    try:
+        logger.info(f"Converting PDF to images: {pdf_path.name}")
+        # pdf2image should respect PDF rotation, but we'll also check EXIF data
+        images = convert_from_path(pdf_path, dpi=200)
+        logger.info(f"Converted {len(images)} pages to images")
+        return images
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)